### ISSUE TITLE
can't add phone in international format

### ISSUE THREAD

**Issue by progreccor** [robotoweb/pagekit-companyinfo#2]
I have tried to add company phone number in international format (like that +7 (925) 832-82-22
And I can't

**Comment by peterbrinck** [comment]
Oh yeah, I haven't thought of that
In Denmark we only use 8 digits, so that's why it's restricted to that

I'll fix it later today

**Comment by progreccor** [comment]
Yes, inside the country we have different types of phone numbers, like this:
8 800-200-55-55 (for bill free numbers)
8 (495) 234-12-12 (for city phones)
and +7 (925) 234-12-12 (for cellular phones)
Please take notice that phone numbers have brackets and dash symbols. And also have spaces.

I have another request for new feature...
Company often have more than one phone number.

They have two or three....
Also in the past year they have viber, telegram  and whatsapp numbers on the company site.

Could you add this fields in your extension?

I think your extension very useful and necessary for every site.

Thank for your effort!

**Comment by progreccor** [comment]
and please add Russian social networks:
vk.com (vkontakte)
and ok.ru (odnoklassniki)
